lib/standard/text/abstract_text.nit

```diff
@@ -1817,6 +1817,12 @@ redef class NativeString
 
 	# Returns `self` as a String of `length`.
 	fun to_s_with_length(length: Int): String is abstract
+
+	# Returns `self` as a String with `bytelen` and `length` set
+	#
+	# SEE: `abstract_text::Text` for more infos on the difference
+	# between `Text::bytelen` and `Text::length`
+	fun to_s_full(bytelen, unilen: Int): String is abstract
```

Comment:
`to_s_with_length` keeps the old terminology as of now, though this could change to avoid future confusion.

`to_s_with_length` actually sets the `bytelen` of the string, `length` is computed lazily